tests/epyccel/test_epyccel_augassign.py

```diff
@@ -0,0 +1,212 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+
+import numpy as np
+import modules.augassign as mod
+
+from pyccel.epyccel import epyccel
+
+
+# += tests
+
+def test_augassign_add_1d(language):
+    f_int     = mod.augassign_add_1d_int
+    f_float   = mod.augassign_add_1d_float
+    f_complex = mod.augassign_add_1d_complex
+    f_int_epyc     = epyccel(f_int, language = language)
+    f_float_epyc   = epyccel(f_float, language = language)
+    f_complex_epyc = epyccel(f_complex, language = language)
+
+    x1_int     = np.zeros(5, dtype=int)
+    x1_float   = np.zeros(5, dtype=float)
+    x1_complex = np.zeros(5, dtype=complex)
+    x2_int     = np.zeros(5, dtype=int)
+    x2_float   = np.zeros(5, dtype=float)
+    x2_complex = np.zeros(5, dtype=complex)
+
+    f_int(x1_int)
+    f_float(x1_float)
+    f_complex(x1_complex)
+    f_int_epyc(x2_int)
+    f_float_epyc(x2_float)
+    f_complex_epyc(x2_complex)
+
+    assert np.array_equal(x1_int, x2_int)
+    assert np.array_equal(x1_float, x2_float)
+    assert np.array_equal(x1_complex, x2_complex)
```

Comment:
Why do you not test the output of the function as well as the argument?

Reply:
I don't think there is a need for that, `np.array_equal` checks the elements of the arrays.

Reply:
But `a` and `b` are not the same object. When everything is working correctly I agree that `np.array_equal` is enough, but if something is broken, having both allows us to tell the difference between a broken `AugAssign` and a broken pointer

Reply:
https://github.com/pyccel/pyccel/pull/1286/commits/7f2400caf3017a9be96cca9df9f72595526e50ac